inductiva/fluids/scenarios/dam_break/dam_break.py

```diff
@@ -52,9 +51,9 @@ def __init__(
     # pylint: disable=arguments-renamed
     def simulate(
         self,
-        simulator: Simulator = DualSPHysics(),
-        resource_pool_id: Optional[UUID] = None,
-        device: Literal["cpu", "gpu"] = "gpu",
+        simulator: Simulator = SPlisHSPlasH(),
```

Comment:
Why are you updating the default simulator in this scenario?

Reply:
I will revert it, it was due to testing.